### DISCUSSION TITLE
Synchronous Exception at 0x00000000BC562000

### DISCUSSION THREAD

**Discussion by ronyt-int** [containers/podman#21122] [General]
after running `podman --log-level=trace machine start` I get `Synchronous Exception at 0x00000000BC562000` in the qemu window. I'm using MacBook Pro M2 Max, Ventura 13.5.

**Comment by 7ojo** [comment]
https://github.com/containers/podman/issues/20776#issuecomment-1874791376

Got the same, and found resolution with using applehv

**Comment by rhatdan** [comment]
My uderstanding on this is that a bad version of QEMU got released to Brew, which is causing these issues.

**Reply by ronyt-int** [reply]
Yes, I installed 8.1 and it works ok, I guess I'll wait for a newer version and try again with that.

**Reply by adrian-moisa** [reply]
Reverting to qemu 8.1.3 works.
